bin/picca_Pk1D.py

```diff
@@ -103,11 +103,19 @@ def process_all_files(index_file_args):
         first_pixel_index = (np.argmax(selected_pixels) if
                              np.any(selected_pixels) else len(selected_pixels))
 
+
+        if linear_binning:
+            max_num_pixels_forest_theoretical=(1180-1050)*(delta.z_qso+1)/pixel_step  #currently no min/max restframe values defined, so hard coding for the moment
```

Comment:
We cannot get the extremum values from the deltas ? It would avoid the hard coding part

Reply:
I think the deltas only store the obs_frame cuts and redshift_cuts, not the restframe range used for the forest. But I'd need to double check. Also not sure if this would create some issues regarding the sidebands...